closure/goog/dom/xml.js

```diff
@@ -39,6 +39,33 @@ goog.dom.xml.MAX_ELEMENT_DEPTH = 256; // Same default as MSXML6.
 
 
 /**
+ * Check for ActiveXObject support by the browser.
+ * @return {boolean} true if browser has ActiveXObject support.
+ * @private
+ */
+goog.dom.xml.hasActiveXObjectSupport_ = function() {
+  try {
+    // Due to lot of changes in IE 9, 10 & 11 behaviour, trying to create the
+    // ActiveXOjbect is a lot more reliable than testing for the existance of
+    // window.ActiveXObject
+    new ActiveXObject('MSXML2.DOMDocument');
```

Comment:
I would find this acceptable, if you write it so property declarations weren't not conditional on the user agent:

/** ... */
goog.dom.xml.ACTIVEX_SUPPORT = goog.userAgent.IE && goog.dom.xml.hasActiveXObjectSupport_();

This would allow both the code removal and avoid the exception on everything but IE.